hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupMerge.java

```diff
@@ -124,4 +128,43 @@ public void TestIncBackupMergeRestore() throws Exception {
     admin.close();
     conn.close();
   }
+
+  @Test
+  public void TestIncBackupMergeRestoreSeparateFs() throws Exception {
+
+    // prepare BACKUP_ROOT_DIR on a different filesystem from HBase
+    File tempDir = new File(FileUtils.getTempDirectory(), UUID.randomUUID().toString());
+    tempDir.deleteOnExit();
```

Comment:
Over the years, we've found that `deleteOnExit` is unreliable for cleaning up unit tests. Better to allocate a path under the maven `target` directory instead. I believe that we have utility methods to accomplish this as part of HBaseTestingUtility.

Reply:
Done.